plugins/nodes/src/js/filters/NodesTextFilter.js

```diff
@@ -0,0 +1,33 @@
+import DSLFilterTypes from "#SRC/js/constants/DSLFilterTypes";
+import DSLFilter from "#SRC/js/structs/DSLFilter";
+
+/**
+ * This filter handles the `text` attributes against service's `getName` value
+ */
+class NodesTextFilter extends DSLFilter {
+  /**
+   * Handle all `name` attribute filters that we can handle.
+   *
+   * @override
+   */
+  filterCanHandle(filterType) {
+    return (
+      filterType === DSLFilterTypes.EXACT || filterType === DSLFilterTypes.FUZZY
+    );
+  }
+
+  /**
+   * Keep only nodes whose name has text
+   *
+   * @override
+   */
+  filterApply(resultset, filterType, filterArguments) {
+    return resultset.filterItems(node => {
+      const hostName = node.getHostName();
```

Comment:
We only want to filter by the `hostName`? Would be cool to have it do a weighted filter of all columns.

Reply:
This could happen in the future if needed. Let us add the filter for now and extend when needed.